android-components/config/detekt.yml

```diff
@@ -784,9 +784,3 @@ style:
     active: true
     excludeImports:
       - 'java.util.*'
-
-# Custom Rules
-
-mozilla-rules:
-  AbsentOrWrongFileLicense:
```

Comment:
I haven't found this rule to work or fail the build in the past. Does the detekt official version work, and does it know to use the right license and format?

(I could pull your patch and test it out to know this, but hoping it's an easy question for you to answer to avoid the extra work)